maquetado/index.html

```diff
@@ -8,37 +8,41 @@
     <link rel="stylesheet" type="text/css" href="../application.css">
 </head>
 <body>
+  <div class="navbar">
```

Comment:
Apply same comments that i made in the PR #8 
You included the changes you made in a different branch too